Tools/plugins/MuonPogTreeProducer.cc

```diff
@@ -369,6 +369,7 @@ void MuonPogTreeProducer::analyze (const edm::Event & ev, const edm::EventSetup
     {
       nGoodMuons = fillMuons(muons,vertexes,beamSpot);
     }
+  eventId_.nMuons = nGoodMuons;
```

Comment:
Hi, 
sorry I am not sure I really understand this. Now we can fill the nGoodMuons requiring a minimum pt, but this is done when producing the ntuples, right? So, if we create the ntuples without any pt cut, we can ask for N number of muons on top, but we can not define the pt of the muons right? So, now, what I do not understand is the vector that Carlo is propossing. Could you explain a little bit more? Maybe I am missing somethign ;)